sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/PrefixesOperationsExtensions.cs

```diff
@@ -18,47 +18,209 @@ namespace Microsoft.Azure.Management.Peering
+                return operations.GetAsync(resourceGroupName, peeringServiceName, prefixName, expand).GetAwaiter().GetResult();
+            }
+
+            /// <summary>
+            /// Gets an existing prefix with the specified name under the given
+            /// subscription, resource group and peering service.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='resourceGroupName'>
+            /// The name of the resource group.
+            /// </param>
+            /// <param name='peeringServiceName'>
+            /// The name of the peering service.
+            /// </param>
+            /// <param name='prefixName'>
+            /// The name of the prefix.
+            /// </param>
+            /// <param name='expand'>
+            /// The properties to be expanded.
+            /// </param>
+            /// <param name='cancellationToken'>
+            /// The cancellation token.
+            /// </param>
+            public static async Task<PeeringServicePrefix> GetAsync(this IPrefixesOperations operations, string resourceGroupName, string peeringServiceName, string prefixName, string expand = default(string), CancellationToken cancellationToken = default(CancellationToken))
+            {
+                using (var _result = await operations.GetWithHttpMessagesAsync(resourceGroupName, peeringServiceName, prefixName, expand, null, cancellationToken).ConfigureAwait(false))
+                {
+                    return _result.Body;
+                }
+            }
+
+            /// <summary>
+            /// Creates a new prefix with the specified name under the given subscription,
+            /// resource group and peering service.
+            /// </summary>
+            /// <param name='operations'>
+            /// The operations group for this extension method.
+            /// </param>
+            /// <param name='resourceGroupName'>
+            /// The name of the resource group.
+            /// </param>
+            /// <param name='peeringServiceName'>
+            /// The name of the peering service.
+            /// </param>
+            /// <param name='prefixName'>
+            /// The name of the prefix.
+            /// </param>
+            /// <param name='peeringServicePrefix'>
+            /// The properties needed to create a prefix.
+            /// </param>
+            public static PeeringServicePrefix CreateOrUpdate(this IPrefixesOperations operations, string resourceGroupName, string peeringServiceName, string prefixName, PeeringServicePrefix peeringServicePrefix)
             {
-                return operations.ListByPeeringServiceAsync(resourceGroupName, peeringServiceName).GetAwaiter().GetResult();
+                return operations.CreateOrUpdateAsync(resourceGroupName, peeringServiceName, prefixName, peeringServicePrefix).GetAwaiter().GetResult();
```

Comment:
Don't we have to pass `CancellationToken` to `CreateOrUpdateAsync`?

Reply:
@AlexanderSher i am not sure? this is auto generated code.